### ISSUE TITLE
Zero-shot evaluation for downstream datasets?

### ISSUE THREAD

**Issue by BIGBALLON** [Shuyu-XJTU/APTM#11]
Firstly, I would like to express my appreciation for the insightful research presented in the paper.

The paper shows the results:

- baseline: training the model **without** pretraining on MALS
- APTM: training the model **with** pretraining on MALS

I am interested in understanding the zero-shot performance, where no further training is performed after pretraining on MALS. Specifically, I would like to see evaluation results on the downstream datasets(CUHK-PEDES, ICFG-PEDES, and RST-PReid):

- zero-shot means: no training, just the model pretraining on MALS, then test on CUHK-PEDES, ICFG-PEDES, and RST-PReid

**Comment by Shuyu-XJTU** [comment]
During pre-training, we used the CUHK-PEDES test set for evaluation, which should provide the experimental results you need. You can try it out.
